ng2/5-piratenameservice/lib/src/name_service.dart

```diff
@@ -12,16 +12,31 @@ class NameService {
   static final Random _indexGen = new Random();
 
   final _names = <String>[
-    'Anne', 'Mary', 'Jack', 'Morgan', 'Roger',
-    'Bill', 'Ragnar', 'Ed', 'John', 'Jane'
+    'Anne',
+    'Mary',
+    'Jack',
+    'Morgan',
+    'Roger',
+    'Bill',
+    'Ragnar',
+    'Ed',
+    'John',
+    'Jane',
   ];
   final _appellations = <String>[
-    'Jackal', 'King', 'Red', 'Stalwart', 'Axe',
-    'Young', 'Brave', 'Eager', 'Wily', 'Zesty'
+    'Jackal',
+    'King',
+    'Red',
+    'Stalwart',
+    'Axe',
+    'Young',
+    'Brave',
+    'Eager',
+    'Wily',
+    'Zesty',
   ];
 
-  String _randomFirstName() =>
-      _names[_indexGen.nextInt(_names.length)];
+  String _randomFirstName() => _names[_indexGen.nextInt(_names.length)];
```

Comment:
A few points (some just FYI):
- That code doesn't appear in the step-5 page.
- We've styled code excerpts so that they scroll horizontally.
- Even when formatted over two lines, this function doesn't really fit unless the browser window is really wide (so the reader would need to scroll anyway).

Regardless, I can make the function wrap as it did before, but I'd rather go with the dartfmt output if we can.